idea/testData/intentions/addBraces/addBracesForSingleStatementIfWithCommentBeneath.kt

```diff
@@ -0,0 +1,10 @@
+fun test() {
+    if (false)<caret>
+        while (true) {
+            foo(1)
+        }
+    //comment about foo(2)
```

Comment:
Could you please add similar test with `else` block, like this
```
if (...) doSmth()
else 
    while (true) {
        foo(1)
    }
//  comment about call below
```

and check how it works now? The code `element.else == null` is a bit suspicious.